internal/data/db_sql.go

```diff
@@ -130,7 +130,12 @@ const sqlFmtFeatures = "SELECT %v %v FROM \"%s\".\"%s\" %v %v %v %s;"
 
 func sqlFeatures(tbl *Table, param *QueryParam) (string, []interface{}) {
 	geomCol := sqlGeomCol(tbl.GeometryColumn, tbl.Srid, param)
-	propCols := sqlColList(param.Columns, tbl.DbTypes, true)
+
+	tblTypes := make(map[string]string)
```

Comment:
Can a copy be avoided here?  Either by caching, or changing the `sqlColList` function?

Reply:
yes I will do it on Monday